.github/ct-lint.yaml

```diff
@@ -1,12 +1,13 @@
-helm-extra-args: --timeout 600s
-check-version-increment: true
-validate_maintainers: false
 debug: true
+charts:
+  - charts/portal
+  - charts/gateway
```

Comment:
do you need to include OTK as well?

Reply:
It's basically the same as the Gateway Chart and not technically supported, we could look to update that in the future.